zerver/lib/message.py

```diff
@@ -196,6 +197,23 @@ def normalize_body(body: str) -> str:
     return truncate_content(body, settings.MAX_MESSAGE_LENGTH, "\n[message truncated]")
 
 
+def normalize_body_for_import(messages: list[Record]) -> list[Record]:
+    for message in messages:
+        content = message["content"]
+        rendered_content = message["rendered_content"]
+        if "\x00" in content:
+            content = re.sub(r"[\x00-\x1F]+", "", content)
+        if "\x00" in rendered_content:
+            rendered_content = re.sub(r"[\x00-\x1F]+", "", rendered_content)
```

Comment:
How did you pick these regular expressions? It looks like you're not just stripping null bytes, and it's unclear to me why you'd be stripping these other characters.

Reply:
These regular expressions were meant to remove characters like `\x01` from Slack's export messages (as referenced in the issue), but they should really only strip null bytes `\x00`. This was an oversight on my part— I’ve corrected it to ensure only null bytes are removed.